variants/esp32s3/ELECROW-ThinkNode-M5/platformio.ini

```
@@ -0,0 +1,22 @@
+[env:thinknode_m5]
+extends = esp32s3_base
+board = ESP32-S3-WROOM-1-N4
+build_flags =
+  ${esp32s3_base.build_flags}
+  -D ELECROW_ThinkNode_M5
+  -I variants/esp32s3/ELECROW-ThinkNode-M5
+  -DEINK_DISPLAY_MODEL=GxEPD2_154_D67
+  -DEINK_WIDTH=200
+  -DEINK_HEIGHT=200
+  -DUSE_EINK
+  -DUSE_EINK_DYNAMICDISPLAY            ; Enable Dynamic EInk
+  -DEINK_LIMIT_FASTREFRESH=10          ; How many consecutive fast-refreshes are permitted    //20
+  -DEINK_LIMIT_RATE_BACKGROUND_SEC=10  ; Minimum interval between BACKGROUND updates          //30
+  -DEINK_LIMIT_RATE_RESPONSIVE_SEC=1   ; Minimum interval between RESPONSIVE updates
+;   -DEINK_LIMIT_GHOSTING_PX=2000        ; (Optional) How much image ghosting is tolerated
+  -DEINK_BACKGROUND_USES_FAST          ; (Optional) Use FAST refresh for both BACKGROUND and RESPONSIVE, until a limit is reached.
+
+lib_deps = ${esp32s3_base.lib_deps}
+  https://github.com/meshtastic/GxEPD2.git  
```

```suggestion
  https://github.com/meshtastic/GxEPD2/archive/1655054ba298e0e29fc2044741940f927f9c2a43.zip
```
Lets use the zip archive :pray: